-------------------------------------------------------------------------------
Test set: Runner.executeRunner
-------------------------------------------------------------------------------
Tests run: 17, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 3.218 sec <<< FAILURE! - in Runner.executeRunner
Then I validate the status code is equals 200(Scenario: Delete existing record and verify the status code)  Time elapsed: 0 sec  <<< FAILURE!
java.lang.AssertionError: expected:<200> but was:<404>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at StepDefinition.TC_API_deleteRequest.i_validate_the_status_code_is_equals(TC_API_deleteRequest.java:34)
	at ?.Then I validate the status code is equals 200(deleteRequest.feature:9)

Scenario: Delete existing record and verify the status code  Time elapsed: 0 sec  <<< FAILURE!
java.lang.AssertionError: expected:<200> but was:<404>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at StepDefinition.TC_API_deleteRequest.i_validate_the_status_code_is_equals(TC_API_deleteRequest.java:34)
	at ?.Then I validate the status code is equals 200(deleteRequest.feature:9)

